openapi/kas-fleet-manager-private.yaml

```diff
@@ -275,19 +275,15 @@ components:
                         type: string
                       principal:
                         type: string
-                      password: 
-                        type: string     
+                      password:
+                        type: string
                 capacity:
                   $ref: "#/components/schemas/ManagedKafkaCapacity"
                 oauth:
                   type: object
                   required:
                     - maximumSessionLifetime
                   properties:
-                    clientId:
-                      type: string
-                    clientSecret:
```

Comment:
@MikeEdgar @shawkins from my understanding, FSO is not using these fields. This PR is removing the field from the database and hence from the ManagedKafka#Spec#Oauth field. Can they safely be removed? Or the fields are current being used in FSO?

Reply:
@machi1990 they've been made optional, but the code is still referencing them and they are present in the API spec. It's safe to no longer send the properties. We can completely remove them in a future FSO change once they're no longer sent.

https://github.com/bf2fc6cc711aee1a0c2a/kas-fleetshard/pull/700

Reply:
Thanks for confirming